ssz/src/main/java/tech/pegasys/teku/ssz/schema/impl/AbstractSszPrimitiveSchema.java

```diff
@@ -60,6 +63,26 @@ public SszDataT createFromBackingNode(TreeNode node) {
     return createFromPackedNode(node, 0);
   }
 
+  @Override
+  public void storeBackingNodes(
+      final TreeNodeStore nodeStore,
+      final int maxBranchLevelsSkipped,
+      final long rootGIndex,
+      final TreeNode node) {
+    nodeStore.storeLeafNode((LeafDataNode) node, rootGIndex);
+  }
+
+  @Override
+  public TreeNode loadBackingNodes(
+      final TreeNodeSource nodeSource, final Bytes32 rootHash, final long rootGIndex) {
+    final Bytes data = nodeSource.loadLeafNode(rootHash, rootGIndex);
```

Comment:
M.b. move `data` calc under `else` block?

Reply:
oh yeah, that's a nice win.